packages/graphiql/src/new-components/List/List.js

```diff
@@ -1,14 +1,14 @@
 /** @jsx jsx */
 import { jsx } from 'theme-ui';
-
 import PropTypes from 'prop-types';
 
-const ListRow = ({ children, flex = false, padding = true }) => (
+const ListRow = ({ children, flex = false, padding = false }) => (
```

Comment:
i like making `padding` explicit, excellent idea

Reply:
i'm a bit conflicted bc ideally itll always be the children of listrow that have padding but it comes in super handy for storybook bc it's mostly text blocks if that makes sense?

Reply:
yeah that does make sense